galley/cmd/galley/cmd/server.go

```diff
@@ -80,13 +80,18 @@ func serverCmd() *cobra.Command {
 				log.Fatalf("Invalid validationArgs: %v", err)
 			}
 
+			galleyStop := make(chan struct{})
+			webhookServerReady := make(chan struct{})
+
 			if serverArgs.EnableServer {
 				go server.RunServer(serverArgs, livenessProbeController, readinessProbeController)
 			}
 			if validationArgs.EnableValidation {
-				go validation.RunValidation(validationArgs, kubeConfig, livenessProbeController, readinessProbeController)
+				go validation.RunValidation(webhookServerReady, validationArgs, kubeConfig,
+					livenessProbeController, readinessProbeController)
+				go validation.ReconcileWebhookConfiguration(webhookServerReady, galleyStop, kubeConfig, validationArgs)
```

Comment:
Please be advised that I am changing the call site here a bit:
https://github.com/istio/istio/pull/15227
Specifically:
https://github.com/istio/istio/pull/15227/files#diff-c9f57ddfffed371a10f24dc5a07fef96R29